### PR TITLE
Feature/11862 fss tooling implement replace access control list

### PR DESCRIPTION
**Pull request by Rashi14748**
This PR contains code changes and unit tests for Replace Acl.

### CODE REVIEW

File: FileShareAdminClient/FileShareApiAdminClient.cs [hunk 1]
```diff
@@ -205,5 +206,19 @@ public async Task RollBackBatchAsync(IBatchHandle batchHandle)
                 response.EnsureSuccessStatusCode();
             }
         }
+
+        public async Task<HttpResponseMessage> ReplaceAclAsync(string batchId, Acl acl)
```
Comment (crdandoukho) [inline FileShareAdminClient/FileShareApiAdminClient.cs:210]: Please add CancellationToken parameter and replace CancellationToken.None

File: Tests/UnitTests/FileShareAdminClientTests/ReplaceAclTests.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            };
+
+            await fileShareApiClient.ReplaceAclAsync(batchId, acl);
+
+           CollectionAssert.AreEqual(new[]
+           {
+                $"PUT:/batch/{batchId}/acl"
+           },
+           
+           lastRequestUris.Select(uri => $"{uri.Item1}:{uri.Item2.AbsolutePath}"));
```
Comment (crdandoukho) [inline Tests/UnitTests/FileShareAdminClientTests/ReplaceAclTests.cs:71]: Please also verify the request body contains the expected ACL values

**Review by crdandoukho — CHANGES_REQUESTED** [review]
Cancellation token should be managed by the client of this library

File: FileShareAdminClient/FileShareApiAdminClient.cs (hunk 1 above)
Comment (Rashi14748) [reply FileShareAdminClient/FileShareApiAdminClient.cs:210]: updated as you suggested, Thanks

File: Tests/UnitTests/FileShareAdminClientTests/ReplaceAclTests.cs (hunk 2 above)
Comment (Rashi14748) [reply Tests/UnitTests/FileShareAdminClientTests/ReplaceAclTests.cs:71]: added Assert for the request body as well. Thanks for review

**Review by crdandoukho — APPROVED** [review]
Thanks Rashi for your changes. 

Looks good

**Review by bhalcha13813 — APPROVED** [review]
Looks good

**Review by jimjamjahaa — APPROVED** [review]

**Pull request merged**